cds_ils/migrator/cli.py

```diff
@@ -110,6 +110,7 @@ def journal(sources, rectype="journal"):
     """Migrate journals from xml dump file."""
     click.echo("Migrating {}s...".format(rectype))
     import_series_from_dump(sources, rectype=rectype)
+    reindex_pidtype("serid")
```

Comment:
it would better to skip indexing, when working with many records at once, the indexing after each command run makes the procedure longer.
You could either remove it or add a flag (--skip-indexing) and make it optional